pysindy/optimizers/stlsq.py

```diff
@@ -140,8 +138,8 @@ def _reduce(self, x, y):
         for _ in range(self.max_iter):
             if np.count_nonzero(ind) == 0:
                 warnings.warn(
-                    """Sparsity parameter is too big ({}) and eliminated all
-                    coefficients""".format(
+                    "Sparsity parameter is too big ({}) and eliminated all "
+                    "coefficients".format(
                         self.threshold
                     )
```

Comment:
I think we'll want to combine this into one line to appease the formatter (black).
```suggestion
                    "coefficients".format(self.threshold)
```